### ISSUE TITLE
DistilBertForSequenceClassification returning nans

### ISSUE THREAD

**Issue by drissg1** [huggingface/transformers#2440]
## DistilBertForSequenceClassification returning NaNs

DistilBertForSequenceClassification using the distilbert-base-uncased is returning Nans for both the logits and loss. Has anyone encountered this issue?

**Comment by sgondala** [comment]
I'm facing this issue. How did you resolve this?

**Comment by Nirvana-2021** [comment]
Me too, and how did you resolve this problem?
